spec/checkMergeConflictSpec.js

```diff
@@ -221,14 +240,68 @@ describe('Merge Conflict Check', () => {
       const params = {
         repo: payloadData.payload.repository.name,
         owner: payloadData.payload.repository.owner.login,
-        number: payloadData.payload.pull_request.number,
+        issue_number: payloadData.payload.pull_request.number,
         name: mergeConflictLabel.name,
       };
       expect(github.issues.removeLabel).toHaveBeenCalledWith(params);
     });
   });
 
-  describe('pull request with unresolved merge conflicts has merge conflict label', () => {
+  describe('pull request with unresolved merge conflicts has merge conflict label and unassigned author', () => {
```

Comment:
Break line, too long.

Reply:
Please check the complete file, to break lines if they are too long.

Reply:
Done.